src/main/java/uk/gov/hmcts/reform/slc/config/PdfConversionConfig.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.slc.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.reform.pdf.generator.HTMLToPDFConverter;
+import uk.gov.hmcts.reform.slc.services.steps.getpdf.IHtmlToPdfConverter;
+
+@Configuration
+public class PdfConversionConfig {
```

Comment:
Why not to take the burden of calling `new` throughout the app and return configured instance of PdfCreator instead?

Reply:
`PdfCreator` is a `@Service` and is injected by Spring.